### PR TITLE
components added: avatar and badge

### PR DESCRIPTION
**Pull request by partha8**
Added avatar and badge components. I also added code snippets.

### CODE REVIEW

**Comment by vrx29** [comment]
looks good to me

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: var(--gap);
+}
+.avatar {
+  object-fit: cover;
+  border-radius: 50%;
+}
+
```
Comment (nayyyhaa) [inline components/avatar/avatar.css]: can make the spacing between CSS classes consistent. Otherwise, good work! 👍

**Review by nayyyhaa — APPROVED** [review]

File: components/avatar/avatar.css (hunk 1 above)
Comment (partha8) [reply components/avatar/avatar.css]: Thanks for reviewing! Made the changes.

**Comment by partha8** [comment]
Got two peer reviews, resolved the issues, and merged into the development branch.

**Pull request merged**